### ISSUE TITLE
Clock don't run when using datetime param is not current time

### ISSUE THREAD

**Issue by phillipet** [furkantektas/analog_clock#6]
```dart 
Widget build(BuildContext context) {
    //DateTime d = DateTime.now();
    DateTime d = DateTime(2020,4,1,1,15,15,15,123);
    return Column(
      children: <Widget>[
        AnalogClock(
          datetime: d,
           isLive: true,
         ),
       ],
    );
...

```
I used another datetime either fetching from server or creating any DateTime the clock don't run
Have you ever seen ?

**Comment by KaeganClark** [comment]
@phong-et I believe the clock repaints itself every second. DateTime.now() Will have a different value every repaint thus giving the appearance of ticking. The example you provide has a set time and every repaint will reflect that provided time.

**Comment by furkantektas** [comment]
Hi @phong-et ,
I am sorry for responding back late. You are correct, the clock behaviour was wrong if you set datetime manually. It's now fixed on 0.1.1.
Thank you for pointing out.
